lib/ops/opsservice/report.go

```diff
@@ -253,17 +256,18 @@ func (s *site) collectEtcdInfoFromMasters(dir string, masters []remoteServer, ru
 	return nil
 }
 
-// collectEtcdInfo collects etcd backup and metrics.
+// collectEtcdInfo collects etcd metrics and captures a snapshot of the data.
 func (s *site) collectEtcdInfo(reportWriter report.FileWriter, runner *serverRunner) error {
-	w, err := reportWriter.NewWriter("etcd.tar.gz")
+	w, err := reportWriter.NewWriter("etcd.json.tar.gz")
```

Comment:
I updated to `etcd.tar.gz` when I added the metrics snapshot. `etcd-backup.json` and `etcd-metrics` are zipped together in `etcd.tar.gz`

Reply:
Thanks for pointing out - I feared I'd break a few things.

Reply:
The thing is is that it remained (or became?) `etcd.json.tar.gz` in `version/7.0.x` [PR](https://github.com/gravitational/gravity/pull/1806/files#diff-9fa9aa39035553b520c7dff230be31a9R262) and I think that's what confused me.

Reply:
Oh, that's my bad. Looks like I didn't update it correctly in 7.0. It was intended to be `etcd.tar.gz`